process/archiving.md

```diff
@@ -0,0 +1,30 @@
+# OpenSSF Project Archiving Process v1.0
```

Comment:
All of this text is include din the README.md

We should only have this text in one place